src/webui/www/private/index.html

```diff
@@ -64,7 +64,7 @@ <h1 class="applicationTitle">qBittorrent Web User Interface <span class="version
                         <a class="returnFalse">QBT_TR(&View)QBT_TR[CONTEXT=MainWindow]</a>
                         <ul>
                             <li><a id="showTopToolbarLink"><img class="MyMenuIcon" src="theme/checked" alt="QBT_TR(&Top Toolbar)QBT_TR[CONTEXT=MainWindow]" width="16" height="16" onload="fixPNG(this)"/>QBT_TR(&Top Toolbar)QBT_TR[CONTEXT=MainWindow]</a></li>
-                            <li><a id="showStatusBarLink"><img class="MyMenuIcon" src="theme/checked" alt="QBT_TR(&Status Bar)QBT_TR[CONTEXT=MainWindow]" width="16" height="16" onload="fixPNG(this)"/>QBT_TR(&Status Bar)QBT_TR[CONTEXT=MainWindow]</a></li>
+                            <li><a id="showStatusBarLink"><img class="MyMenuIcon" src="theme/checked" alt="QBT_TR(Status &Bar)QBT_TR[CONTEXT=MainWindow]" width="16" height="16" onload="fixPNG(this)"/>QBT_TR(Status &Bar)QBT_TR[CONTEXT=MainWindow]</a></li>
```

Comment:
I remember that you have a different opinion on this, but I would still have separate translations for the GUI and Web UI, otherwise we will mess around with such unmaintainable things.

Reply:
Maintaining by hand `extra_translations.h` with many strings will become a hassle too. (If we separate translations)

Reply:
>but I would still have separate translations for the GUI and Web UI

I hope you don't wish to do it in this PR...
One possible downside is many duplicate translation strings will be created.

>otherwise we will mess around with such unmaintainable things.

Another way is make these strings (the ones with memonic) live in extra_translation.h.

Reply:
>I hope you don't wish to do it in this PR...

Not here, of course.
>One possible downside is many duplicate translation strings will be created.

WebUI is almost a separate thing. The fact that it is now following desktop GUI is only a special case. IMO, this is not the best design idea for the web interface (especially in the modern world).

Reply:
@Chocobo1, what about the last commit? At least its message is confusing:
```
Also removed `static` specifier for variables in header, it could
cause multiple instantiations for each include of this header.
```

Reply:
>what about the last commit? At least its message is confusing:

Description removed.

Reply:
About the duplicate strings: I know that lupdate can recognize same strings and puts only one entry in the .ts pointing to the different line numbers. However, I think this is limited to the same context.

Future script to extract strings from WebUI sources is welcome. Probably it will be easier if the script extracted the strings to `extra_translation.h` and then we run lupdate.

Reply:
>Probably it will be easier if the script extracted the strings to extra_translation.h and then we run lupdate.

The one more goal of separate WebUI translation is to have it really separate (i.e. in its own files) so we could allow alternative WebUI to use translation mechanism too.